pkg/apis/core/validation/validation.go

```diff
@@ -264,6 +264,15 @@ func ValidateRuntimeClassName(name string, fldPath *field.Path) field.ErrorList
 	return allErrs
 }
 
+// validateOverhead can be used to check whether the given Overhead is valid.
+func validateOverhead(overhead core.ResourceList, fldPath *field.Path) field.ErrorList {
+	//reuse the ResourceRequirements validation logic
+	resourceReq := core.ResourceRequirements{
+		Limits: overhead,
+	}
+	return ValidateResourceRequirements(&resourceReq, fldPath)
```

Comment:
It would be better to remove the resourceReq with the following code which would make the code more concise.

```go
	return ValidateResourceRequirements(&core.ResourceRequirements{
		Limits: overhead,
	}, fldPath)

```

Reply:
[1]